corehq/form_processor/backends/sql/dbaccessors.py

```diff
@@ -434,6 +434,33 @@ def iter_forms_by_last_modified(start_datetime, end_datetime):
             load_source='forms_by_last_modified'
         )
 
+    @staticmethod
+    def iter_forms_by_last_modified_in_domain(domain, start_datetime, end_datetime):
```

Comment:
this doesn't seem to be used anywhere

Reply:
Yeah I was using it at one point but switched back to ES. I'll leave it for now in case I decide to go back to querying from the accessors, but will remove it if I end up merging this. In reality, this is a long way from being merge worthy. 

I'll close this out and make a draft PR so that I don't need to bug multiple reviewers when code is ready to be reviewed.